addons/crm/report/crm_opportunity_report_views.xml

```diff
@@ -1,6 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
+        <record id="crm_opportunity_report_view_tree" model="ir.ui.view">
```

Comment:
CM: being picky here, it is a bit confusing. I thought you wanted to remove the buttons from the pipeline tree view, but it is only for reporting. Can you maybe reword to "remove reporting pipeline tree buttons"  - ish ?